src/envoy/transcoding/config.cc

```diff
@@ -134,10 +134,6 @@ Config::Config(const Json::Object& config) {
         default:
           break;
       }
-
-      //      pmb.Register("POST", "/" + service->full_name() + "/" +
```

Comment:
if you don't have options in the proto, GetExtension() will just return a default instance of HttpRule, which is fine.